src/helpers/apify_helper.js

```diff
@@ -18,24 +18,24 @@ const DEFAULT_POOLING_INTERVAL_MILLIS = 5000;
 /**
  * Asynchronously waits until run is finished
  */
-async function waitUntilRunFinished(runId, actId, apifyClient, interval = DEFAULT_POOLING_INTERVAL_MILLIS) {
-    let running = true;
-    let actRun;
+async function waitUntilRunFinished(runId, apifyClient, interval = DEFAULT_POOLING_INTERVAL_MILLIS) {
```

Comment:
Why not use directly https://github.com/apify/apify-client-js/blob/master/src/resource_clients/run.ts#L132 now?